locales-hi-IN.xml

```diff
@@ -0,0 +1,361 @@
+      <single>भाग</single>
+      <multiple>parts</multiple>
+    </term>
+    <term name="section">
+      <single>अनुभाग</single>
+      <multiple>sections</multiple>
+    </term>
+    <term name="sub verbo">
+      <single>sub verbo</single>
+      <multiple>sub verbis</multiple>
+    </term>
+    <term name="verse">
+      <single>पद</single>
+      <multiple>verses</multiple>
+    </term>
+    <term name="volume">
+      <single>वॉल्यूम</single>
+      <multiple>वॉल्यूम्ज़</multiple>
+    </term>
+
+    <!-- SHORT LOCATOR FORMS -->
+    <term name="book" form="short">
+      <single>bk.</single>
+      <multiple>bks.</multiple>
+    </term>
+    <term name="chapter" form="short">
+      <single>chap.</single>
+      <multiple>chaps.</multiple>
+    </term>
+    <term name="column" form="short">
+      <single>col.</single>
+      <multiple>cols.</multiple>
+    </term>
+    <term name="figure" form="short">
+      <single>fig.</single>
+      <multiple>figs.</multiple>
+    </term>
+    <term name="folio" form="short">
+      <single>fol.</single>
+      <multiple>fols.</multiple>
+    </term>
+    <term name="issue" form="short">
+      <single>no.</single>
+      <multiple>nos.</multiple>
+    </term>
+    <term name="line" form="short">
+      <single>l.</single>
+      <multiple>ll.</multiple>
+    </term>
+    <term name="note" form="short">
+      <single>n.</single>
+      <multiple>nn.</multiple>
+    </term>
+    <term name="opus" form="short">
+      <single>op.</single>
+      <multiple>opp.</multiple>
+    </term>
+    <term name="page" form="short">
+      <single>पृ.</single>
+      <multiple>पृ.</multiple>
+    </term>
+    <term name="number-of-pages" form="short">
+      <single>पृ. स.</single>
+      <multiple>पृ. स.</multiple>
+    </term>
+    <term name="paragraph" form="short">
+      <single>para.</single>
+      <multiple>paras.</multiple>
+    </term>
+    <term name="part" form="short">
+      <single>pt.</single>
+      <multiple>pts.</multiple>
+    </term>
+    <term name="section" form="short">
+      <single>sec.</single>
+      <multiple>secs.</multiple>
+    </term>
+    <term name="sub verbo" form="short">
+      <single>s.v.</single>
+      <multiple>s.vv.</multiple>
+    </term>
+    <term name="verse" form="short">
+      <single>v.</single>
+      <multiple>vv.</multiple>
+    </term>
+    <term name="volume" form="short">
+      <single>vol.</single>
+      <multiple>vols.</multiple>
+    </term>
+
+    <!-- SYMBOL LOCATOR FORMS -->
+    <term name="paragraph" form="symbol">
+      <single>¶</single>
+      <multiple>¶¶</multiple>
+    </term>
+    <term name="section" form="symbol">
+      <single>§</single>
+      <multiple>§§</multiple>
+    </term>
+
+    <!-- LONG ROLE FORMS -->
+    <term name="director">
+      <single>निर्देशक</single>
+      <multiple>directors</multiple>
+    </term>
+    <term name="editor">
+      <single>सम्पादक</single>
+      <multiple>editors</multiple>
```

Comment:
@neha-sri, if necessary we can accept a partially translated CSL locale file, but is there a reason why you haven't translated some of the terms? (like the plural forms here)

Reply:
So I wanted to consult a few people and then submit the remaining words as a follow up change after trying this with a possible style. I can submit it on this PR also if thats preferable.

Reply:
I'll just merge it for now (it looks fine otherwise). Thanks, and we're looking forward to a more complete version!